airflow/config_templates/config.yml

```diff
@@ -682,6 +682,16 @@
       type: string
       example: ~
       default: "http://localhost:8080"
+    - name: default_ui_timezone
+      description: |
+        Default timezone to display all dates in the UI in, can be UTC, system,
+        or any IANA timezone string (e.g. Europe/Amsterdam). If left empty the
+        default vaule of core/default_timezone will be used
```

Comment:
```suggestion
        Default timezone to display all dates in the UI, can be UTC, system,
        or any IANA timezone string (e.g. Europe/Amsterdam). If left empty the
        default value of core/default_timezone will be used
```

Reply:
Yup, will push this up (plus .cfg) once the current tests run finishes.